packages/esm-commons-lib/src/components/card-summary/summary-card.component.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import styles from './summary-card.scss';
+import { SkeletonText, Tile, Column } from '@carbon/react';
+import { LazyCell } from '../lazy-cell/lazy-cell.component';
+import { OpenmrsEncounter } from '@openmrs/openmrs-form-engine-lib';
+import { fetchLatestEncountersOfTypes } from './helpers';
+
+export interface GeneralSummaryCardProps {
+  patientUuid: string;
+  columns: Array<SummaryCardColumn>;
+  headerTitle: string;
+}
+
+export interface SummaryCardColumn {
+  key: string;
+  header: string;
+  encounterTypes: string[];
+  getObsValue: (encounter: OpenmrsEncounter[]) => string | Promise<string>;
+  getObsSummary?: (encounter: OpenmrsEncounter[]) => string | Promise<string>;
+}
+
+export const GeneralSummaryCard: React.FC<GeneralSummaryCardProps> = ({ patientUuid, columns, headerTitle }) => {
```

Comment:
Was wondering where this component was cause i assumed the name based off the file name. Wouldn't it be better to align the 2?

Reply:
Renamed to `SummaryCard`